Makefile

```diff
@@ -1,9 +1,15 @@
-
 DOCKER_COMPOSE=docker-compose.yml
 DOCKERFILE=Dockerfile
 DOCKER_REGISTRY=ghcr.io
 DOCKER_REPOSITORY=${DOCKER_REGISTRY}/ix/sandbox
 HASH_FILES=requirements*.txt package.json Dockerfile
+
+# check for md5sum or md5 for hashing
+HASHER := $(shell command -v md5sum 2> /dev/null)
+ifndef HASHER
+    HASHER := md5 -r
+endif
+
 IMAGE_TAG=$(shell cat $(HASH_FILES) | md5sum | cut -d ' ' -f 1)
```

Comment:
but then need to use HASHER in the line below right?

Reply:
Ah whoops. You are correct that it wasn't actually being used.  Should be fixed in 5e14b9a